edm4hep.yaml

```
@@ -619,62 +619,168 @@ datatypes:
     Members:
       - float weight                        // weight of this association
     OneToOneRelations:
-     - edm4hep::ReconstructedParticle  rec  // reference to the reconstructed particle
-     - edm4hep::MCParticle sim              // reference to the Monte-Carlo particle
+     - edm4hep::ReconstructedParticle  from  // reference to the reconstructed particle
+     - edm4hep::MCParticle to              // reference to the Monte-Carlo particle
+    ExtraCode:
+      declarationFile: "edm4hep/extra_code/AssociationGetters.h.ipp"
+      implementationFile: "edm4hep/extra_code/AssociationGetters.cc.ipp"
+    MutableExtraCode:
+      declaration: "
+      [[deprecated(\"use setFrom instead\")]]\n
+      void setRec(const edm4hep::ReconstructedParticle& rec);\n
+      [[deprecated(\"use setSim instead\")]]\n
```

I believe it should hint to use `setTo` instead of `setSim`. It's repeated a few times in the file
```suggestion
      [[deprecated(\"use setTo instead\")]]\n
```

---

Ah yes, copy&past error. Thanks